### ISSUE TITLE
Default template config in webspace is ignored

### ISSUE THREAD

**Issue by TimGroenenberg** [sulu/sulu#5055]
| Q | A
| --- | ---
| Sulu Version | 2.0.4
| Chrome Version 79.0.3945.130 | Chrome Version 79.0.3945.130

# Actual Behavior
The default-templates in the webspace config is ignored when creating a new page.

# Expected Behavior

When configuring the default template for the `page` type, I expect this template to be selected automatically when creating a new page.

# Steps to Reproduce

Add the following config in your webspace file:
```xml
    <default-templates>
        <default-template type="page">default</default-template>
        <default-template type="home">homepage</default-template>
    </default-templates>
```
Create extra page templates in your project.
Create a new page in Sulu, a different template is selected then the default.

**Comment by alexander-schranz** [comment]
@danrot do you know why this happens now?

**Comment by danrot** [comment]
Don't know right now, but I will have a look at it.

**Comment by danrot** [comment]
Ok, now I remember the problem, but what is confusing to me is that the current solution should never have worked :thinking: 

The problem is that we have to [use the `webspaceStore` in the `TemplateToolbarAction`](https://github.com/sulu/sulu/blob/e17397b/src/Sulu/Bundle/PageBundle/Resources/js/views/Form/toolbarActions/TemplateToolbarAction.js#L14), which is an asynchronous action...

This has already bothered me in multiple place, because you have take care of storing the webspace on your own... I am thinking about moving that (and maybe also the localizations from the `LocalizationStore`) to our config request, which is already localized as well. Then we could use the store to store the data and just [filling it in an initializer](https://github.com/sulu/sulu/blob/edbf2d5/src/Sulu/Bundle/SecurityBundle/Resources/js/index.js#L17). Then we wouldn't have to bother with asynchronity anymore, but it would be a JS BC break, and cause changes in multiple places.

The alternative would be to make the `ToolbarActions` returning a promise, or at least introduce a `loading` flag, and make sure that they are called multiple times. Which would also be a BC break, while thinking about it :thinking:

**Comment by alexander-schranz** [comment]
This is fixed in #5119 and will be in the next release.
